Plugins/Wox.Plugin.BrowserBookmark/Main.cs

```diff
@@ -48,7 +50,7 @@ public List<Result> Query(Query query)
             return returnList.Select(c => new Result()
             {
                 Title = c.Name,
-                SubTitle = "Bookmark: " + c.Url,
+                SubTitle = context.API.GetTranslation("wox_plugin_browserbookmark_bookmark") + ": " + c.Url,
```

Comment:
This part of the description will be removed per https://github.com/jjw24/Wox/pull/51 so translation for this one will not be required